_includes/samples/PriceFeeds/PriceConsumerV3.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"log"
+
+	proxy "github.com/aelmanaa/chainlink-price-feed-golang/aggregatorv3"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func main() {
+	client, err := ethclient.Dial("https://kovan.infura.io/v3/<infura_project_id>")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Price Feed address
+	addr := "0x9326BFA02ADD2366b30bacB125260Af641031331"
```

Comment:
since the address is not being passed in as an arg we can combine lines 20 and 21 and make the variables more descriptive:
`aggregatorAddress = common.HexToAddress("0x9326BFA02ADD2366b30bacB125260Af641031331")